XCUITest/fa.lproj/InfoPlist.strings

```diff
@@ -2,5 +2,5 @@
 "CFBundleName" = "$(PRODUCT_NAME)";
 
 /* (No Commment) */
-"CFBundleShortVersionString" = "۱.۰";
+"CFBundleShortVersionString" = "۱٫۰";
```

Comment:
I don't know what this is. This looks like a really weird version string. It is for a Test, so maybe it doesn't matter.